### ISSUE TITLE
The page titles for the 'hashpassword.html' and 'opml.html' are not getting inherited from the configs.

### ISSUE THREAD

**Issue by akash07k** [SSilence/selfoss#1147]
The titles of above aaid pages are not getting inheritid from the configurations.
I've tried modifying somethings in 'selfoss-ui.js' but it didn't worked.
For example:
        if ($('body').is('#hashpasswordbody, #opmlbody')) {
            // we do not want to create UI for non-app pages but we want to inherit the title of those pages from the configurations.
            $(document).attr('title', selfoss.htmlTitle);
            return;
        }

But it is not working.
I've tried various more things but no luck as of now.
@jtojnar 
Can you please look into this?

**Comment by jtojnar** [comment]
Yeah, we no longer include the whole selfoss script to those pages. I intend to integrate the OPML page into the selfoss settings page, which will fix it for that, and also allow us to translate it. For the password page, it should be only used by the person who installs selfoss, so they will know their instance is runnng selfoss and it should not be confusing.

In d39d5503, I have made the titles more descriptive, which is even more important.

**Comment by akash07k** [comment]
ok, yes, making the page titles more descriptive and contextual is
more important

**Comment by akash07k** [comment]
I was wondering that isn't there any way to retrieve the titles from
the configurations as previous?
